addons/account/models/chart_template.py

```diff
@@ -172,25 +183,47 @@ def _notify_accountant_managers(taxes_to_check):
             partner_ids=partner_managers_ids,
         )
 
+    def _validate_taxes_country(chart_template, template2tax):
+        """ Checks that existing taxes' country are either compatible with the company's
+        fiscal country, or with the chart_template's country.
+        """
+        for company_id in template2tax:
+            company = env['res.company'].browse(company_id)
+            for template_id in template2tax[company_id]:
+                tax = env['account.tax'].browse(template2tax[company_id][template_id])
+                if not any([
+                    chart_template.country_id and tax.country_id == chart_template.country_id,
+                    tax.country_id == company.account_fiscal_country_id
+                ]):
```

Comment:
```suggestion
                if (not chart_template.country_id or tax.country_id != chart_template.country_id) and tax.country_id != company.account_fiscal_country_id:
```
Would be more simple in this case